src/main/java/org/folio/services/kafka/InventoryProducerRecordBuilder.java

```diff
@@ -51,7 +51,7 @@ public InventoryProducerRecordBuilder propagateOkapiHeaders(Map<String, String>
 
   public KafkaProducerRecord<String, String> build() {
     try {
-      var valueAsString = getMapper().writeValueAsString(this.value);
+      var valueAsString = value instanceof String ? (String) value : getMapper().writeValueAsString(value);
```

Comment:
Is there a better way to handle this?

I'm surprised that there isn't a mapper that writes a string as a string that can avoid this special case workaround.

Reply:
I couldn't find such a mapper.